MAUI/Cartesian-Charts/StackedLine.md

```diff
@@ -0,0 +1,445 @@
+        ItemsSource = viewModel.Data4,
+        ShowMarkers= true, 
+    };
+
+    chart.Series.Add(series1);
+    chart.Series.Add(series2);     
+    chart.Series.Add(series3); 
+    chart.Series.Add(series4); 
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Stacked Line Marker support in MAUI Chart](Chart-types-images/stacked_line_marker_support.png)
+
+### Marker customization
+
+In order to change the series markers appearance, create an instance of the [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html#Syncfusion_Maui_Charts_StackingLineSeries_MarkerSettings) property. The following properties are used to customize marker appearance.
+
+* [Type](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Type), of type `ShapeType`, describes the shape of the series marker. The default value of this property is [ShapeType.Circle]().
+* [Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Stroke), of type `Brush`, indicates the brush used to paint the marker border.
+* [StrokeWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_StrokeWidth), of type `double`, indicates the width of the marker border.
+* [Fill](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Fill), of type `Brush`, indicates the color of the marker.
+* [Width](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Width), of type `double`, indicates the width of the marker.
+* [Height](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Height), of type `double`, indicates the height of the marker.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+
+        <chart:SfCartesianChart.BindingContext>
+            <local:ViewModel/>
+        </chart:SfCartesianChart.BindingContext>
+        ...
+        <chart:StackingLineSeries ItemsSource="{Binding Data1}"
+                                  XBindingPath="Month"
+                                  YBindingPath="Value"
+                                  ShowMarkers="True">
+            <chart:StackingLineSeries.MarkerSettings>
+                <chart:ChartMarkerSettings Type="Diamond" 
+                                           StrokeWidth="2"
+                                           Fill="White"
+                                           Height="15"
+                                           Width="10"/>
+            </chart:StackingLineSeries.MarkerSettings>    
+        </chart:StackingLineSeries>   
+    
+        <chart:StackingLineSeries ItemsSource="{Binding Data2}"
```

Comment:
add single series like this
![image](https://github.com/user-attachments/assets/b6712513-a351-4fea-87d6-e2476009af21)

Reply:
modified code